### DISCUSSION TITLE
Serverless Devs 社区月会

### DISCUSSION THREAD

**Discussion by anycodes** [Serverless-Devs/Serverless-Devs#454] [Meeting]
**会议名称**：Serverless Devs 社区月会

**会议背景**：Serverless Devs 作为 Serverless 领域的开源开放的工具链平台，致力于为广大的开发者提供 Serverless 应用全生命周期管理工具，让开发者可以像玩转手机一样玩转 Serverless，为了更好的带动社区氛围，与广大开发者一同开发出最适合广大开发者的工具，Serverless Devs 社区决定每个月进行一次月会。

**会议频率**：每月一次

**会议时间**：原则上是每个月第一周的星期四晚上7:00-8:30社区月会，如果是休息日则顺延一周
  - 2022年4月7日 晚上7:00-8:30

**会议内容**：
1. 会议前一天，会在当前`discussions`下进行更新会议内容，所以可以点击右侧的`Notifications`进行提醒；
2. 主持人简单开场【每月从社区中选取主持人，有兴趣的可以参与报名】
3. 主持人介绍一下工具在过去一个月的更新，并做一下演示
4. 讨论环节
     - 吐槽：主持人，社区相关同学，进行工具相关的吐槽，所有在线的同学可以开麦参与吐槽；
     - 规划：主持人，社区相关同学，对社区当月的工作做规划，所有在线的同学可以提出意见；
5. 会议结束
6. 会议结束第二天，在Github公开相关的会议内容

**会议地址**：钉钉会议
```
主题：Serverless Devs 社区月会
时间：04月07日 周四 19:00 - 20:30
入会口令：180 446 39235
电话呼入：+8657128356289（中国大陆）+862281944271（中国大陆）
入会链接：dingtalk://dingtalkclient/page/videoConfFromCalendar?confId=20ae7aea-a8f3-4c9b-8e87-735b563e48ff&calendarId=1674493036
```
> 大家也可以加入钉钉群`32116533`随时间进行沟通和讨论。

**主持人轮班制度**：
1. 主持人必须是社区的贡献者，社区贡献的文档可以参考[CONTRIBUTING.md](https://github.com/Serverless-Devs/Serverless-Devs/blob/master/CONTRIBUTING.md)，社区贡献者列表参考[CONTRIBUTORS.md](https://github.com/Serverless-Devs/Serverless-Devs/blob/master/CONTRIBUTORS.md)，如果您为社区贡献了代码、文档、组件、应用、插件都可以将您的信息添加到社区贡献者列表参考[CONTRIBUTORS.md](https://github.com/Serverless-Devs/Serverless-Devs/blob/master/CONTRIBUTORS.md)，可以通过提PR的方法，补充完整`Add contributor`选项即可；
2. 想要申请主持人，可以加钉钉群`32116533`咨询，或者发送邮件到社区邮箱`postmaster@devsapp.cn`进行申请；申请时请提供自己的`Github账号 || Github邮箱`，我们会邀请进入Org的社区Team；
3. 社区主持人理论上是轮班制度，每个月换一个主持人，如果有特殊情况可以在Team中进行沟通；

**Comment by anycodes** [comment]
# 202204 月度会议

**会议时间** ：2022年4月7日 晚上7:00-8:30

**主持人** ： @anycodes 

**会议内容**：
- 开场，简单介绍 Serverless Devs 的社区会议情况：@anycodes 
- Serverless Devs 最新功能演示： @heimanba 
- Serverless Devs 吐槽会：所有在会议的同学全员讨论
- Serverless Devs 本月规划：所有在会议的同学全员讨论
- 主持人总结，会议结束： @anycodes 

**会议地址**：
```
入会口令：180 446 39235
电话呼入：+8657128356289（中国大陆）+862281944271（中国大陆）
入会链接：dingtalk://dingtalkclient/page/videoConfFromCalendar?confId=20ae7aea-a8f3-4c9b-8e87-735b563e48ff&calendarId=1674493036
```

**Reply by heimanba** [reply]
先吐个槽：Serverless-Devs 确实蛮强大的，但是看到最佳实践， 和step by step教程，需要开发者自己去探索
